src/bitcoin-cli.cpp

```diff
@@ -1101,6 +1101,68 @@ static UniValue GetNewAddress()
     return ConnectAndCallRPC(&rh, "getnewaddress", /* args=*/{}, wallet_name);
 }
 
+/**
+ * IsExclusivelyCliCommand checks if a string is a cli-command that needs to run exclusively
+ * @param[in] cli_command  Reference to const string cli-command.
+ * @returns true if cli_command it's included in the set of the exclusively_commands.
+ */
+bool IsExclusivelyCliCommand(const std::string cli_command)
+{
+    // Initialize set with cli-commands that run exclusively
+    const std::set<std::string> exclusively_commands{"-generate", "-netinfo", "-addrinfo", "-getinfo"};
```

Comment:
I'd prefer to keep the definition as is since the function is called only once, thanks for the suggestion.

Reply:
Sorry, not sure what you mean here. It looks to me like this function is called for every argument passed to bitcoin-cli, and therefore this set will be initialized for every argument instead of just once.

Reply:
Right, sorry, I haven't expressed myself correctly, I meant that the validation was only "used" once but you are right, the `IsExclusivelyCliCommand` function is called for every argument passed to `bitcoin-cli`, still, `static` would keep the set  throughout the lifetime of the program.